### PR TITLE
Fix dependencies up to work with the latest Ratpack

### PR DESCRIPTION
**Pull request by ksimons**

### CODE REVIEW

File: src/main/java/com/schibsted/ratpack/datadog/DataDogMetricsFactory.java [hunk 1]
```diff
@@ -5,18 +5,19 @@
...
 
-    private Boolean enabled;
+    private Boolean enabled = false;
 
     public Boolean getEnabled() {
         return enabled;
     }
 
     public void setEnabled(Boolean enabled) {
+        System.out.println("setting " + (enabled == null));
```
Comment (erigus6) [inline src/main/java/com/schibsted/ratpack/datadog/DataDogMetricsFactory.java]: Don't use System.out.println use a logger and use it in buildModule

Comment (ksimons) [inline src/main/java/com/schibsted/ratpack/datadog/DataDogMetricsFactory.java]: Hehe, that was debugging code. Will kill it.

Comment (ksimons) [inline src/main/java/com/schibsted/ratpack/datadog/DataDogMetricsFactory.java]: Fixed.

**Comment by erigus6** [comment]
LGTM

**Pull request merged**